sites/cheerpj/src/content/docs/11-guides/basic-server-setup.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Basic Server Setup
+description: How do serve your own Java application with CheerpJ
+---
+
+## Overview
+
+To run your Java applications in the browser, you will need to host the application files. This guide will cover some simple HTTP servers that can be used for basic testing. We will also discuss NGINX which is useful in more advanced scenarios, for example when self hosting CheerpJ.
+
+> [!note] Important
+> The content of this guide is specifically made for testing setups and should not be used for production environments. In production environments it is essential to utilize HTTPS, both to improve security and to gain access to advanced Web APIs.
+
+## Static "one-line" servers
```

Comment:
"one-line", One-line _One-Line_ (used in various sentences)

Let's settle on a single style